### PR TITLE
[Doc Improvement][Respond to the dialog submit action][3499111]

### PR DESCRIPTION
**Pull request by v-vanisreeth**
Changed the link to open message extensions.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/426cdfd42ac62c839c31bbcbca8ec5304be82485/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 87 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93646795-f228-496b-88e6-d2cdf74993f5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/51ef65ff6961c9661ca8f5d68473c6b60194eb83/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/14abc336-a6c2-41e8-b275-37036d95d163) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ff0ae8613ba56e60e866c24df0080236f6df8027/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 92 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0b3718ed-4d85-4c46-89bf-9da1c188c9d5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50686599b30b5b37c259b5e7c90b0a5732dac36b/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/07b5ce4c-888e-452b-9e08-50635b19a91b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6434c4c1c448dea4da78f496a1292ef4b5aab5f5/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 89 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5048cee3-c9ac-45b9-a11f-11a321861038) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dd377785881ad3edf40072261c45ca666225592d/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 88 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0591605c-a8be-4626-938a-f4fe18c1ef2f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a24bd7c3d15cd5c70f7a49e91c9d2bb668e32fc5/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4867fb39-5297-4eda-87f3-2b0382d6a776) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d20c73de5f0ff05ff4e1457312a9dcadc9c9738f/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/734a0e67-fee0-4679-8e4a-c1dca5b08dcd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/21ffd78a09e0d514209191d1417f4451e40266d5/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 92 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/608fbcb4-c002-4e43-accd-af31ec98433a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md [hunk 1]
```diff
@@ -599,8 +597,8 @@ The following section is a description of the entities in the `OnBehalfOf` Array
 |Field|Type|Description|
 |:---|:---|:---|
 |`itemId`|Integer|Describes identification of the item. Its value must be `0`.|
-|`mentionType`|String|Describes the mention of a "person". |
-|`mri`|String|Message resource identifier​ (MRI) of the person on whose behalf the message is sent. Message sender name would appear as "\<user\> through \<bot name\>". |
+|`mentionType`|String|Describes the mention of a "person." |
```
Comment (ChetanSharma-msft) [inline msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md:600]: It should be "person".

**Review by ChetanSharma-msft — APPROVED** [review]
Added one small comment.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ae5e76730a03a024647663ae17c00c621c369ed5/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe70115e-8284-4b9e-b122-7e57841158a3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md (hunk 1 above)
Comment (v-vanisreeth) [reply msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md]: I've changed it.

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc9351b2798911c29b3fcad3419674a6c4c919ec/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 91 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b8c33aec-6a9f-402d-82f0-4701cc20bcbb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**